examples/kubernetes/block-volume/specs/pod.yaml

```diff
@@ -0,0 +1,18 @@
+
+apiVersion: v1
+kind: Pod
+metadata:
+  name: app 
+spec:
+  containers:
+  - name: app 
+    image: busybox 
+    command: ["/bin/sh", "-c"]
+    args: ["tail -f /dev/null"]
+    volumeDevices:
+    - name: data
+      devicePath: /dev/xvda
+  volumes:
+  - name: data
+    persistentVolumeClaim:
+      claimName: block-claim
```

Comment:
add the new line in the end and in all the files wherever required

Reply:
fixed